ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -547,8 +741,14 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
               Cancel
             </Button>
             <TooltipTrigger content={[<ul>Missing Required Fields:
-              { this.isEmpty('name') && <li> Name </li> }
-              { this.isEmpty('description') && <li> Description </li> }
+              { this.isEmpty(this.state.workspace, 'name') && <li> Name </li> }
+              { this.isEmpty(this.state.workspace.researchPurpose, 'intendedStudy') &&
+              <li>Field of intended study</li>}
+              { this.isEmpty(this.state.workspace.researchPurpose, 'anticipatedFindings') &&
+              <li>Anticipated findings</li>}
+              { this.isEmpty(this.state.workspace.researchPurpose, 'softwareChoice') &&
+              <li>Reason for choosing AoU</li>}
+              { !this.categoryIsSelected() && <li>Research focus</li>}
```

Comment:
Should this also validate and error on "Yes I am focusing on a specific population, but no I have not selected which one"?

Reply:
good thought